pro/src/pages/Home/VenueOfferSteps/VenueOfferSteps.tsx

```diff
@@ -128,26 +130,49 @@ const VenueOfferSteps = ({
                     Créer une offre
                   </ButtonLink>
                 )}
-                {hasMissingReimbursementPoint && (
-                  <ButtonLink
-                    className={styles['step-button-width']}
-                    isDisabled={!hasVenue}
-                    variant={ButtonVariant.BOX}
-                    icon={fullNextIcon}
-                    link={{
-                      to: `/structures/${offererId}/lieux/${venueId}#reimbursement`,
-                      isExternal: false,
-                    }}
-                    onClick={() => {
-                      logEvent?.(VenueEvents.CLICKED_VENUE_ADD_RIB_BUTTON, {
-                        venue_id: venueId || '',
-                        from: OFFER_FORM_NAVIGATION_IN.HOME,
-                      })
-                    }}
-                  >
-                    Renseigner des coordonnées bancaires
-                  </ButtonLink>
-                )}
+                {!isNewBankDetailsJourneyEnabled &&
```

Comment:
Pour info quand est-ce que ce FF sera enlevé ? il a l'air de pas mal complexifier les conditions

Reply:
Oui c'est clair malheureusement on n'a pas trop le choix, il sera supprimé quelques ité après la MEP de la fonctionnalité